services/zone-service.ts

```diff
@@ -213,6 +215,80 @@ class ZoneService {
         return interestZone.unsterilizedCats.id(catID);
     }
 
+    private async findImage(imageID: string) {
```

Comment:
for image related functions (public or private), please use as dedicated service.